src/clr-angular/index.ts

```diff
@@ -21,5 +21,7 @@ export * from './accordion/stepper/index';
 export * from './progress/spinner/index';
 export * from './progress/progress-bars/index';
 
+export { FOCUS_ON_VIEW_INIT } from './utils/focus/focus-on-view-init/focus-on-view-init.provider';
```

Comment:
Can we just export this through the utils directory instead of explicitly here? We want to avoid this.

Reply:
Yeah, I think we can do that.